source/configuration/modules/imuxsock.rst

```diff
@@ -294,8 +334,8 @@ The following sample is used activate message annotation and thus
 trusted properties on the system log socket. module(load="imuxsock" #
 needs to be done just once SysSock.Annotate="on")
 
-Legacy Configuration Parameters
--------------------------------
+|FmtObsoleteDescription|
```

Comment:
@deoren I did not have time to read up more intensely on RST: would it be possible to do a replacement of a complete text block? I mean we need to have the header and some introductory text be the same for all module descriptions. Would be great if we could get that from a single place...

Reply:
@rgerhards I'm not sure if a complete block is supported, but I should be able to test sometime today to see if I can figure it out.

- http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#substitution-definitions
- http://docutils.sourceforge.net/docs/ref/rst/directives.html#include

Include statements are supported. Perhaps mixing the two would give the desired effect. 

That said, if it's the header syntax that you're having problems with, the number of characters beneath the text (or substitution reference) might be the issue. Literal text:

```
Legacy Configuration Parameters
-------------------------------
```

you would instead match the characters beneath to match the length of the substitution reference, not the text it will be expanded to when replaced:

```
|FmtObsoleteDescription|
------------------------
```

It took me a while to figure that out.

Reply:
@rgerhards I've not found a way to bundle both the substitution text AND the header characters into a single substitution entry. Using an `include` file is the only way I know of to pull that off.

Is what you did with the substitution reference and header here a concern?

https://github.com/rgerhards/rsyslog-doc/commit/06b484a0e34549266ed02627acba9f8ee94a4050#diff-e80eae1b47ef8f45a205d11832bf4ffdR337

```
|FmtObsoleteDescription|
------------------------
```

That looks to be easy enough to maintain going forward. If we move the header level, we adjust the header characters out for the appropriate syntax. If we want to make sure that ALL module docs are consistent with header levels (something I was going to focus on after some other updates), then we could create an include file and use that, but wouldn't that be overkill here?